index.html

```diff
@@ -1,33 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" type="text/css" href="styles.css" />
-  <title>Portfolio setup and mobile first</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <title>Portfolio setup and mobile first</title>
 </head>
+
 <body>
-    <header>
-        <nav class="navbar">
-            <a class=  "box1" href="https://www.github.com/AlejandraKeber">Alejandra</a>
-            <a class="box2" href="#Menu"><img src="./Images/Navbarmenu.png" alt="Menu"></a>
-        </nav>
-    </header>
+    <div class="wrapper">
+        <header>
+            <nav class="navbar">
+                <a class="box1" href="https://www.github.com/AlejandraKeber">Alejandra</a>
+                <a class="box2" href="#Menu"><img src="./Images/Navbarmenu.png" alt="Menu"></a>
+                <ul class="box3">
+                    <li><a href="#Portfolio">Portfolio</a></li>
+                    <li><a href="#About">About</a></li>
+                    <li><a href="#Contact">Contact</a></li>
+                </ul>
+            </nav>
+        </header>
+
         <main>
-            <section>
+            <section class="main">
                 <div>
                     <h1 class="headline">I'm Alejandra! <br>Glad to see you!</h1>
-                    <p class="headlinecont">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    <p class="headlinecont">I’m a software developer! I can help you build a product , feature or
```

Comment:
Great job @AlejandraKeber 🤝

- [ ] Kindly fix the background image of your main page. It appears to be overstretched which makes your design differ from the one provided in the Figma template.
- Your design
![Your main](https://user-images.githubusercontent.com/63917725/186829640-13cdc5c8-d981-46bb-acd0-4149b95895dd.PNG)

- Figma design
![Figma main](https://user-images.githubusercontent.com/63917725/186829692-ed743b07-de97-428a-8014-238c695373d6.PNG)